packages/roosterjs-editor-core/lib/undo/UndoSnapshots.ts

```diff
@@ -7,25 +14,20 @@ const MAXSIZELIMIT = 1e7;
 /**
  * A class to help manage undo snapshots
  */
-export default class UndoSnapshots implements UndoSnapshotsService {
-    private snapshots: string[];
-    private totalSize: number;
-    private currentIndex: number;
+export default class UndoSnapshots implements UndoSnapshotsService, Snapshots {
+    snapshots: string[] = [];
+    totalSize = 0;
+    currentIndex = -1;

```

Comment:
nit: Types were removed for `totalSize `and `currentIndex`

Reply:
yes. type is not necessary here